README.md

```diff
@@ -34,29 +34,27 @@ This repository uses [Yarn](https://yarnpkg.com/getting-started/install) to mana
 In order to install them, please, run the following commands.
 
 ```sh
-cd onchain
+cd onchain/rollups
 yarn install
 ```
 
 ## Tests
 
 If you plan to run the [Forge](https://book.getfoundry.sh/getting-started/installation) tests, there still some setup left to do.
-Assuming you are on the `onchain` directory, and that [Docker Engine](https://docs.docker.com/get-docker/) is running on the background, you may run the following commands.
+Assuming you are on the `onchain/rollups` directory, and that [Docker Engine](https://docs.docker.com/get-docker/) is running on the background, you may run the following command.
+This command will build the Cartesi Machine image necessary to build the proofs.
```

Comment:
should we say "Cartesi Virtual Machine"?

Reply:
This is actually a great topic of discussion. I personally prefer "Cartesi Machine".
https://discord.com/channels/600597137524391947/1169240346564251750